LedgerSMB/Database/Change.pm

```diff
@@ -0,0 +1,280 @@
+=head1 NAME
+
+LedgerSMB::Database::Change - Database change scripts for LedgerSMB
+
+=cut
+
+package LedgerSMB::Database::Change;
+
+use strict;
+use warnings;
+use Digest::SHA;
+use Cwd;
+
+our $reloading = 0;
+
+=head1 SYNOPSIS
+
+my $dbchange = LedgerSMB::Database::Change->new(path => $path,
+                                      properties => $properties);
+
+my $content = $dbchange->content()
+my $sha = $dbchange->sha();
+my $content_wrapped = $dbchange->content_wrap($before, $after);
+
+=head1 METHODS
+
+=head2 constructor
+
+LedgerSMB::Database::Change->new($path, $properties);
+
+$properties is optional and a hashref with any of the following keys set:
+
+=over
+
+=item no_transactions
+
+Do not wrap this in a transaction
+
+=item reload_subsequent
+
+If this one has changed, then reload further modules
+
+=back
+
+=cut
+
+sub new {
```

Comment:
Yeah.  there really isn't enough at present to justify one.  I might switch and go with Moo(se) if more functionality is added but for now it seemed like too small a data structure to really benefit from it.